packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -78,13 +76,45 @@ const SupportListItem = ({ item }: { item: SupportItem }) => {
   );
 };
 
-export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
-  const { items } = useSupportConfig();
+const createDisplayedItems = (
+  showCondition: Boolean | undefined,
+  generalItems: SupportItem[],
+  supportItems: SupportItem[] | undefined,
+) => {
+  if (showCondition && supportItems) {
+    return supportItems;
+  }
+  if (supportItems) {
+    return [...supportItems, ...generalItems];
+  }
+  return generalItems;
+};
+
+type Props = {};
 
+type SupportButtonProps = {
+  title?: string;
+  supportItems?: SupportItem[];
+  hideDefaultItems?: boolean;
+};
+
+export const SupportButton = ({
+  supportItems,
+  title,
+  hideDefaultItems = false,
+  children,
```

Comment:
Should this be marked as deprecated by the new `title` prop or do we want to support both moving forward? Removing it it now, might break any plugins using this output the repo.

Reply:
@andrewthauer  We could support both, for now, to don't break anything and when you have to up a major version, we could remove the `children` prop

Reply:
@andrewthauer do you think it would be better remove `title` prop?

Reply:
I share the same thought, but as you said, if I remove children that could break things to another person, that is why I decide keep both props, but I don't know if this is enough for the approval

Reply:
I think having both makes sense and you could add a `/* @deprecated */` jsdoc to indicate it will be removed at a later date. I personally think having both in the long term might be confusing as it's not clear what content should go where.